client/views/resetpwd/resetpwd.js

```diff
@@ -0,0 +1,103 @@
+import $ from 'jquery';
+
+Accounts.onResetPasswordLink((token, done) => {
+  Router.go('resetpwd');
+});
+
+if (Accounts._resetPasswordToken) {
+  Session.set('resetPasswordVar', Accounts._resetPasswordToken);
+}
+
+function showChangePwdError(reason, parentElement, nextElement) {
+  if (typeof currentError !== 'undefined') {
+    Blaze.remove(currentError);
+  }
+  const parentNode = $('.' + parentElement)[0];
+  const nextNode = $('.' + nextElement)[0];
+  currentError = Blaze.renderWithData(Template.form_error, reason, parentNode, nextNode);
+}
+
+Template.resetpwd.onRendered(() => {
+  $('.formcontainer').hide().fadeIn(400);
+  $('#darker').hide().fadeIn(400);
+});
+
+Template.resetpwd.helpers({
+  resetPassword() {
+    return Session.get('resetPasswordVar');
+  },
+});
+
+Template.resetpwd.events({
+  'keypress #newpasswordconfirm': function (event, template) {
+    event.which = event.which || event.keyCode;
+    if (event.which === 13) {
+      event.preventDefault();
+      $('#newpwdsubmitbutton').click();
+    }
+  },
+  'keypress #loginemail': function (event, template) {
+    event.which = event.which || event.keyCode;
+    if (event.which === 13) {
+      event.preventDefault();
+      $('#forgotsubmitbutton').click();
+    }
+  },
+  'click #forgotsubmitbutton': function (event, template) {
+    $('#mailSentMessage').css('display', 'none');
+    const email = $('#loginemail').val();
+    $('#newPwdLoadingSpinner').css('display', 'block');
+    Accounts.forgotPassword({ email: email }, function (e) {
+      $('#newPwdLoadingSpinner').css('display', 'none');
+      if (e) {
+        const errorMessage = [
+          { reason: 'User not found', message: "The email address that you've entered doesn't match any account." },
+          { reason: 'Internal server error', message: 'Please check your internet connection.' },
+        ];
+        const in1 = errorMessage.findIndex(x => x.reason === e.reason);
+        if (in1 === -1) {
```

Comment:
This could be more readable as `if (Object.keys(errorMessage).includes(e.reason)) {` (and then switch the condition).